warehouse/accounts/views.py

```diff
@@ -1514,13 +1520,17 @@ def default_response(self):
         return {
             "pending_github_publisher_form": self.pending_github_publisher_form,
             "pending_google_publisher_form": self.pending_google_publisher_form,
+            "pending_activestate_publisher_form": self.pending_activestate_publisher_form,  # noqa
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** use the code we're disabling so that we have a better idea ofwhy a rule was disabled.‏

```suggestion
            "pending_activestate_publisher_form": self.pending_activestate_publisher_form,  # noqa: E501
```

Similar note applies to other `noqa` instances.

Reply:
Done this in a local change.  Nice catch.  Didn't notice that used elsewhere in the code.  I like this much better!